### PR TITLE
feature- Added my profile to linkFree

### PR DESCRIPTION
**Pull request by Moeen-Ul-Islam**
Hi I am Moeen, I just added my profile to the linkFree, please review it!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/338"></a>

### CODE REVIEW

File: public/data/Moeen-Ul-Islam.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "url": "https://twitter.com/moeenulislam_",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://linkedin.com/in/moeenulislam",
+        "icon": "linkedIn"
+      }
+    ]
+  }
```
Comment (CodingSpecies) [inline public/data/Moeen-Ul-Islam.json:22]: I don't think we need a the brace to be tabbed @Moeen-Ul-Islam . Thanks.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/Moeen-Ul-Islam.json (hunk 1 above)
Comment (Panquesito7) [inline public/data/Moeen-Ul-Islam.json:22]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Panquesito7** [comment]
Looks like your profile is already on the list. I suggest closing this PR.

**Comment by Moeen-Ul-Islam** [comment]
> Looks like your profile is already on the list. I suggest closing this PR.

It is not merged yet!

**Comment by Panquesito7** [comment]
It's already in the repository https://github.com/EddieHubCommunity/LinkFree/blob/main/public/data/Moeen-Ul-Islam.json

**Pull request closed without merging**